src/slices/releaseSlice.js

```diff
@@ -0,0 +1,71 @@
+import { createSlice } from "@reduxjs/toolkit";
+import Axios from "axios";
+
+// First, define the reducer and action creators via `createSlice`
+export const releaseSlice = createSlice({
+  name: "releases",
+  initialState: { pending: false, error: "", items: [] },
+  reducers: {
+    getPending(state, action) {
+      state.pending = true;
+    },
+    getSuccess(state, action) {
+      state.pending = false;
+      state.items = action.payload;
+    },
+    getError(state, action) {
+      state.pending = false;
+      state.error = action.payload;
+    },
+    putIsPublicPending: state => {
+      state.pending = true;
+    },
+    putIsPublicSuccess: (state, action) => {
+      state.pending = false;
+      const newValue = action.payload.data.isPublic;
+      state.items.find(obj => {
+        if (obj.id == action.payload.id) {
+          obj.isPublic = newValue;
+        }
+      });
+    },
+    putIsPublicError: (state, action) => {
+      state.pending = false;
+      state.error = action.payload;
+    }
+  }
+});
+
+// Destructure and export the plain action creators
+export const {
+  getPending,
+  getSuccess,
+  getError,
+  putIsPublicPending,
+  putIsPublicSuccess,
+  putIsPublicError
+} = releaseSlice.actions;
+
+// Thunk for fetching
+export const fetchReleases = () => async dispatch => {
+  dispatch(getPending());
+  Axios.get("/releases")
+    .then(res => {
+      dispatch(getSuccess(res.data));
+    })
+    .catch(err => dispatch(getError(err)));
+};
+
+// Thunk for updating isPublic
+export const updateIsPublic = (id, isPublic) => async dispatch => {
+  dispatch(putIsPublicPending());
```

Comment:
Den her e unødvending om den skal gjer det samme som `getPending()`

Reply:
E ikke formålet med "pending" at man bruk dem for å gjør nå nyttig for brukern? Som feks å gi en indikasjon på at alle releases lastes inn, eller at isPublic e på tur å gå igjønnom. Trur egentlig at i tilfellet med at vi send request på å ender isPublic, som e en ganske "lett" operasjon og vent på svar, ikke treng nå indikasjon på "pending" men heller error.